articles/backup/backup-azure-vms-encryption.md

```diff
@@ -153,6 +153,7 @@ Encrypted VMs can only be restored by restoring the VM disk as explained below.
 Restore encrypted VMs as follows:
 
 1. [Restore the VM disk](backup-azure-arm-restore-vms.md#restore-disks).
+  + Note: After restoring the VM disk swap the OS Disk of the Original VM with the restored VM disk without having to recreate it again learn more: https://azure.microsoft.com/en-in/blog/os-disk-swap-managed-disks/ +
```

Comment:
Please use the following format with correct indentation. Before you merge, please send the updated section in WORD for review (attach the file).

>[!NOTE]
>After restoring the VM disk, swap the OS Disk of the original VM with the restored VM disk without recreating it. [Learn more](https://azure.microsoft.com/en-in/blog/os-disk-swap-managed-disks/).